src/Core/Checkout/Customer/Validation/Constraint/CustomerZipCode.php

```diff
@@ -31,31 +32,42 @@ class CustomerZipCode extends Constraint
      *
      * @deprecated tag:v6.8.0 - Parameter $options will be required and natively typed as array
```

Comment:
this deprecation is not changed and the information that the other parameters will be required is missing

Reply:
because they're not required atm

next major it will be

```
public function __construct(protected ?string $countryId = null, protected bool $caseSensitiveCheck = true)
```

Reply:
okay, but is still says `Parameter $options will be required and natively typed as array
` which is wrong, right?